pkgs/misc/vim-plugins/update.py

```diff
@@ -407,6 +407,12 @@ def parse_args():
         default=DEFAULT_OUT,
         help="Filename to save generated nix code",
     )
+    parser.add_argument(
+        "--debug",
```

Comment:
How about `--sync` ?

Reply:
You can mention in the help test that this is useful for debugging.

Reply:
Done.